optionals/compat_adr_97/stringtable.xml

```diff
@@ -0,0 +1,221 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project name="ACE">
+    <Package name="ace_compat_adr_90">
```

Comment:
Should be `Compat_ADR_97`.

And every Key ID should be `STR_ACE_Compat_ADR_97_*`.